datafusion/src/physical_plan/parquet.rs

```diff
@@ -59,6 +61,8 @@ use crate::datasource::datasource::{ColumnStatistics, Statistics};
 use async_trait::async_trait;
 use futures::stream::{Stream, StreamExt};
 
+use super::SQLMetric;
```

Comment:
The changes to this file are to add metrics on the pruning (that are then used in the test)